src/Connector.sol

```diff
@@ -134,14 +134,14 @@ contract CentrifugeConnector {
         memberlist.updateMember(user, validUntil);
     }
 
-    function handleTransfer(uint64 poolId, bytes16 trancheId, address user, uint256 amount) public onlyRouter {
+    function handleTransfer(uint64 poolId, bytes16 trancheId, address user, uint128 amount) public onlyRouter {
```

Comment:
I thought we agreed on having `u256` as the amount or is this not related to the message part?

Reply:
We didn't agree on having `u256` tho, we agreed on `u128` since that's the type that is supported on Centrifuge chain, the core product. We can always coerce `uint128` to `u256` safely (for Ethereum integrations) but we can't coerce `u256` to `u128`. The only way around this would be by changing the `Balance` type in Centrifuge to `u256`.

What are your thoughts on this?

Reply:
Are, then I was wrong. No that is perfectly fine. Had just another memory. Just need to remember to adapt the spec ^^

Reply:
Cool :)